src/index.html

```diff
@@ -15,8 +15,103 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;500;900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header class="container">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo header"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="link">
+            <a
+              href="index.html"
+              class="nav--link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="index.html"
+              class="nav--link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="index.html"
+              class="nav--link"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li
+            class="link"
+            data-qa="hover"
+          >
+            <a
+              href="index.html"
+              class="nav--link"
+              data-qa="hover"
+            >
+              LAPTOPS &AMP; COMPUTERS
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="index.html"
+              class="nav--link"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="index.html"
+              class="nav--link"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="index.html"
+              class="nav--link"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="index.html"
```

Comment:
The `href` attributes for the navigation links are all set to 'index.html'. These should be updated to actual URLs or intended targets for each navigation item to function correctly.